PurchasesCoreSwift/Logging/Logger.swift

```diff
@@ -29,20 +29,59 @@ import Foundation
         guard level != .debug || shouldShowDebugLogs else { return }
         NSLog("[\(frameworkDescription)] - \(level.description()): \(message)")
     }
+    
+    static func log(level: LogLevel, intent: LogIntent, message: String) {
```

Comment:
@aboedo Error from CircleCI: "cannot find type 'LogIntent' in scope" but I'm able to compile and run this code on a simulator. This also happened when this function was an `@objc public static func`.

Reply:
`LogIntent.swift` is in the file system and in the repo, but it hasn't been added to the `.xcproject` file, so xcode can't find it. Dragging and dropping it from Finder to the project in Xcode should fix it, same for `Emoji.swift`. 
Make sure they get added to the target, i.e.:
<img width="284" alt="image" src="https://user-images.githubusercontent.com/3922667/102361336-e6356980-3f91-11eb-9f47-7697b0c31685.png">

Reply:
maybe the changes to the `.xcproject` files have been done but not committed? it'd explain why it works for you locally